src/Symfony/Bridge/PhpUnit/Tests/ExpectDeprecationTraitTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
+
+final class ExpectDeprecationTraitTest extends TestCase
+{
+    use ExpectDeprecationTrait;
+
+    /**
+     * Do not remove this test in the next major version.
+     *
+     * @group legacy
+     */
+    public function testOne()
+    {
+        $this->expectDeprecation('foo');
+        @trigger_error('foo', E_USER_DEPRECATED);
+    }
+
+    /**
+     * Do not remove this test in the next major version.
+     *
+     * @group legacy
+     */
+    public function testMany()
+    {
+        $this->expectDeprecation('foo');
+        $this->expectDeprecation('bar');
```

Comment:
To actually test that the feature is working, we would also need to write tests where `expectDeprecation` is called but not deprecation is triggered, to show that the test fail in such case.

Of course, this must not be part of the main Symfony testsuite run (here, you are writing tests as if they were testing `trigger_error`, not as if they were testing the bridge), but as phpt tests which can assert that running these tests report a failure.